app/src/components/v-upload.test.ts

```diff
@@ -0,0 +1,111 @@
+import type { GlobalMountOptions } from '@/__utils__/types';
+import { mount } from '@vue/test-utils';
+import { expect, test, beforeEach, vi, describe } from 'vitest';
+import { createI18n } from 'vue-i18n';
+import { Tooltip } from '../__utils__/tooltip';
+import { createTestingPinia } from '@pinia/testing';
+import { setActivePinia } from 'pinia';
+
+import VUpload from './v-upload.vue';
+
+const i18n = createI18n({
+	legacy: false,
+	messages: {
+		'en-US': {
+			click_to_browse: 'Click to Browse',
+			drag_file_here: 'Drag & Drop a File Here',
+			import_from_url: 'Import File from URL',
+			folders: 'Folders',
+		},
+	},
+});
+
+const global: GlobalMountOptions = {
+	stubs: [
+		'v-icon',
```

Comment:
I didn't stub `v-button` because it wouldn't allow me to upload the file. But without the stub, it does return a warning.  Not sure if I should remove the upload test, and just keep the snapshot test?

```
[Vue warn]: Failed to resolve component: v-button
If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement.
  at <VUpload multiple=false preset=undefined fileId="unique_id"  ... >
  at <VTUROOT>
  ```